azure-iot-hub/azure/iot/hub/protocol/models/fault_injection_connection_properties.py

```diff
@@ -1,28 +0,0 @@
-# coding=utf-8
-# --------------------------------------------------------------------------
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
-# --------------------------------------------------------------------------
-
-from msrest.serialization import Model
-
-
-class FaultInjectionConnectionProperties(Model):
```

Comment:
What's up with all these deleted files?  I don't see any swagger changes that would explain this.

Reply:
Not sure. I tried generating the code both on Ubuntu 20.04 through WSL and on Windows and both produce the same result.